platform/android/export/export.cpp

```diff
@@ -1341,6 +1356,46 @@ class EditorExportPlatformAndroid : public EditorExportPlatform {
 			return str;
 		}
 	}
+
+	// Creates strings.xml files inside the gradle project for different locales.
+	void _create_project_name_strings_files(const Ref<EditorExportPreset> &p_preset) {
+		XMLParser parser;
+		const char *locales[] = {
+			"ar", "bg", "ca", "cs", "da", "de", "el", "en", "es", "es_ES", "fa", "fi", "fr", "hi",
+			"hr", "hu", "in", "it", "iw", "ja", "ko", "lt", "lv", "nb", "nl", "pl", "pt", "ro",
+			"ru", "sk", "sl", "sr", "sv", "th", "tl", "tr", "uk", "vi", "zh", "zh_HK", "zh_TW"
+		};
+		// TODO: use package/name or application/config/name?
+		String package_name = p_preset->get("package/name");
+		String project_name = get_project_name(package_name);
+		size_t num_locales = sizeof(locales) / sizeof(locales[0]);
+
+		// Creates and stores the default strings.xml file.
+		String default_xml_string = "<?xml version=\"1.0\" encoding=\"utf-8\"?>\n"
```

Comment:
Could you change this to a format string and then use vformat to fill in the `godot_project_name_string` field?
ex: 
````
String xml_string_base = "<?xml version=\"1.0\" encoding=\"utf-8\"?>\n"
"<resources>\n\t<string name=\"godot_project_name_string\">%s</string>\n</resources>"; 
String default_xml_string = vformat(xml_string_base, project_name);
````

That way you could define the format string only once rather than duplicating it on line 1387.